src/PhpDocument.php

```diff
@@ -53,7 +53,7 @@ class PhpDocument
      *
      * @var Node[]
      */
-    private $stmts;
+    private $stmts = [];
```

Comment:
I don't think this is good - there is a semantic difference between "a document has no statements" and "we couldn't parse the statements"